src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/tasks/BlobProcessorTask.java

```diff
@@ -183,16 +183,15 @@ private void processZipFile(
         if (leaseId.isPresent()) {
             // Zip file will include metadata.json and collection of pdf documents
             try (ZipInputStream zis = loadIntoMemory(cloudBlockBlob, zipFilename)) {
-                registerEvent(ZIPFILE_PROCESSING_STARTED, container.getName(), zipFilename, null);
+                handleEventRelatedError(ZIPFILE_PROCESSING_STARTED, container.getName(), zipFilename, null, null);
```

Comment:
`ZIPFILE_PROCESSING_STARTED` is not an error afaik

Reply:
this will be updated to just use `EnvelopeProcessor::createEvent` in the following PR(s) ;). Don't want to change any extra namings/logic here

Reply:
squashed that local change as it was not too big of a change and still related to event creations